### PR TITLE
Change the signature name for the geocoder street point function

### PR DESCRIPTION
**Pull request by ethervoid**
In order minimize confusion, we have changed the function `cdb_geocode_street_point_v2` to `cdb_geocode_street_point`. This change introduces a drop statement in the migration file to remove a previous function that was never used.

### CODE REVIEW

**Comment by ethervoid** [comment]
retest this please

**Comment by ethervoid** [comment]
retest this please

**Comment by ethervoid** [comment]
retest this please

**Comment by ethervoid** [comment]
@rafatower CR this please

File: client/cdb_geocoder_client--0.1.0--0.0.1.sql [hunk 1]
```diff
@@ -1,5 +1,8 @@
+DROP FUNCTION IF EXISTS cdb_geocoder_client.cdb_geocode_street_point (text, text, text, text);
+DROP FUNCTION IF EXISTS cdb_geocoder_client._cdb_geocode_street_point (text, text, text, text);
+-- This functions could exists due a first deploy with this name
 DROP FUNCTION IF EXISTS cdb_geocoder_client.cdb_geocode_street_point_v2 (text, text, text, text);
-DROP FUNCTION IF EXISTS cdb_geocoder_client._cdb_geocode_street_point_v2 (text, text, text, text);
+DROP FUNCTION IF EXISTS cdb_geocoder_client._cdb_geocode_street_point_v2 (text, text, text, text, text, text);
```
Comment (rafatower) [inline client/cdb_geocoder_client--0.1.0--0.0.1.sql]: I'd just remove these two lines.

Comment (ethervoid) [inline client/cdb_geocoder_client--0.1.0--0.0.1.sql]: why?

**Comment by rafatower** [comment]
:+1:

**Comment by rafatower** [comment]
we will have to force the reinstallation of the client extension on new users, right? otherwise it will do nothing.

**Comment by ethervoid** [comment]
I've done it after the deploy. There were new 70 users.

**Pull request merged**